extensions/2.0/Khronos/KHR_xmp/README.md

```diff
@@ -0,0 +1,337 @@
+| [ResourceRef](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#resourceref)      | Object, we refer to the XMP documentation for a description of its properties |
+| [URL](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#url)                      | String formatted as described in the XMP documentation |
+| [Rational](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#rational)            | String formatted as described in the XMP documentation |
+| [FrameRate](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#framerate)          | String formatted as described in the XMP documentation |
+| [FrameCount](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#framecount)        | String formatted as described in the XMP documentation |
+| [Part](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#part)                    | String formatted as described in the XMP documentation |
+
+### Additional Properties
+
+| XMP Property Type | JSON Type |
+| ----------------- | --------- |
+| [ResourceEvent](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/ResourceEvent.md)| Object, we refer to the XMP documentation for a description of its properties |
+
+## Defining XMP Metadata
+
+An indirection level is introduced to avoid replicating the same XMP metadata in multiple glTF nodes.
+XMP metadatas are defined within a dictionary property in the glTF scene description file by adding an `extensions` property to the top-level glTF 2.0 object and defining a `KHR_xmp` object. The `KHR_xmp` object defines two properties:
+
+* `namespaces` : a dictionary mapping XMP [namespaces](https://github.com/adobe/xmp-docs/blob/master/Namespaces.md) to the URI where they are defined.
+* `packets`: an array of metadata packets referencing the namespaces defined in the `namespaces` dictionary.
+
+The following example defines a glTF scene with a sample XMP metadata.
+
+```json
+"extensions": {
+    "KHR_xmp" : {
+        "namespaces" : {
+            "dc" : "http://purl.org/dc/elements/1.1/"
+        },
+        "packets" : [
+            {
+                "dc:contributor" : [ "Creator1Name", "Creator2Email@email.com", "Creator3Name<Email@email.com>"],
+                "dc:coverage" : "Bay Area, California, United States",
+                "dc:creator" : [ "CreatorName", "CreatorEmail@email.com"],
+                "dc:date" : ["1997-07-16T19:20:30+01:00"],
+                "dc:description" : { "en-us": "text"},
+                "dc:format" : "model/gltf-binary",
+                "dc:identifier" : "urn:stock-id:292930",
+                "dc:language" : ["en"],
+                "dc:publisher" : ["Company"],
+                "dc:relation" : ["https://www.khronos.org/"],
+                "dc:rights" : {"en-us": "BSD"},
```

Comment:
I believe they aren't part of RFC 3066, so I think the answer is not supported. I will ask Leonard for confirmation.

Reply:
In the JSON-LD serialization of XMP (ISO 16684-3), you use `i-default` instead of `x-default` since (as noted in an earlier comment) we now use [BCP-47](https://tools.ietf.org/html/bcp47) codes and so that's `i-default`

Reply:
I'm not sure what this implies for the `KHR_xmp` extension, sorry. As an implementor, where would I find this information? 😕